src/main.rs

```diff
@@ -31,6 +31,10 @@ struct Cli {
     #[clap(long, hide = true)]
     generate_pdf_metadata: bool,
 
+    /// Generate a JSON schema for the configuration file.
+    #[clap(long, hide = true)]
```

Comment:
I don't know if I'd `hide = true` this, this should be visible for people doing a `--help`

Reply:
Ah, nice catch. Probably a copy paste mistake, now fixed.